api/spec/v1.yaml

```diff
@@ -1933,6 +1970,61 @@ components:
         self_delegation_shares:
           allOf: [$ref: '#/components/schemas/TextBigInt']
           description: The shares of tokens this validator has delegated to itself, and are NOT in the process of debonding.
+        active_balance_24:
+          allOf: [$ref: '#/components/schemas/TextBigInt']
+          description: The active_balance of this validator account 24 hours ago.
+        num_delegators:
+          type: integer
+          format: uint64
+          description: The number of accounts that have delegated token to this account.
+    
+    EscrowHistory:
```

Comment:
Was also considering naming this `ValidatorHistory` and `ValidatorHistoryPoint` below; open to suggestions